diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -38,7 +77,9 @@ class LineLength : Rule("line-length") {
 
         val configuration = LineLengthConfiguration(
                 configRules.getRuleConfig(LONG_LINE)?.configuration ?: mapOf())
+
         if (node.elementType == FILE) {
+            positionByOffset =calculateLineColByOffset(node.text)
```

Comment:
No, it was a good idea to initialize this thing only once we find a long line: most files won't have any violations, so calculating `calculateLineColByOffset` for every file will be quite a waste of resources. If I understand correctly, check `::positionByOffset.isInitialized` which exists in kotlin for `lateinit` properties, isn't very costly and initialization can be performed only when we find a long line and if property isn't already initialized.